test/TestProjects/Models-TypeSpec/src/Generated/ModelsTypeSpecClient.cs

```diff
@@ -1022,7 +1022,7 @@ public virtual Response<OutputModel> FixedFloatEnum(FixedFloatEnum input, Cancel
         /// </item>
         /// <item>
         /// <description>
-        /// Please try the simpler <see cref="FixedFloatEnumAsync(FixedFloatEnum,CancellationToken)"/> convenience overload with strongly typed models first.
+        /// Please try the simpler <see cref="FixedFloatEnumAsync(Models.FixedFloatEnum,CancellationToken)"/> convenience overload with strongly typed models first.
```

Comment:
`Models` namespace is needed here because there is method with the same name - `FixedFloatEnum`.